drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -136,8 +141,16 @@ def status
       Types::HmisSchema::Enums::EnrollmentStatus.from_enrollment(object)
     end
 
+    def client
+      load_ar_association(object, :client)
+    end
+
+    def household
+      load_ar_association(object, :household)
+    end
+
     def household_size
-      load_ar_association(object, :household)&.household_size
+      household&.household_size
```

Comment:
Not for this PR, but I maybe we should add a column for `household_size` to the `hmis_households` view. This `household_size` method will join the Enrollment table to find out the size. (The view already does a join like that, too, so I guess it's happening twice? I'm not sure.)

Reply:
Actually in this case, since our base object is an Enrollment, it could make more sense to just to self-join on HouseholdID and data_source_id. Something like a `has_many :household_enrollments` on the Enrollment model. Not sure exactly how to set it up in rails but I think it would be faster. Just capturing my thoughts while I'm here, but no action needed :)